src/views/wallet/WalletViewNav.tsx

```diff
@@ -1,23 +1,26 @@
 import React from 'react'
 
 import { Menu } from 'antd'
-import { Link } from 'react-router-dom'
+import { Link, useLocation } from 'react-router-dom'
 
 import * as walletRoutes from '../../routes/wallet'
+import { StyledMenu } from './WalletViewNav.style'
 
 export const WalletViewNav: React.FC = (): JSX.Element => {
+  const location = useLocation()
+
   return (
-    <Menu mode="horizontal">
-      <Menu.Item key={0}>
+    <StyledMenu mode="horizontal" selectedKeys={[location.pathname]}>
+      <Menu.Item key="/wallet/assets">
```

Comment:
This key is unsafe. Imagine if we will change the route for `wallet/assets`, that's why we have `walletRoutes`. Take the tabs of `HeaderComponent` as an example how to deal with keys and routes.